interface/usergroup/usergroup_admin.php

```diff
@@ -351,8 +359,8 @@
             "', mname = '"         . add_escape_custom(trim((isset($_POST['mname']) ? $_POST['mname'] : ''))) .
             "', lname = '"         . add_escape_custom(trim((isset($_POST['lname']) ? $_POST['lname'] : ''))) .
             "', suffix = '"         . add_escape_custom(trim((isset($_POST['suffix']) ? $_POST['suffix'] : ''))) .
-            "', google_signin_email = '" . add_escape_custom(trim((isset($_POST['google_signin_email']) ? $_POST['google_signin_email'] : ''))) .
-            "', valedictory = '"         . add_escape_custom(trim((isset($_POST['valedictory']) ? $_POST['valedictory'] : ''))) .
+            "', google_signin_email = " . add_escape_custom($googleSigninEmail) .
+            ", valedictory = '"         . add_escape_custom(trim((isset($_POST['valedictory']) ? $_POST['valedictory'] : ''))) .
```

Comment:
looks like something with a missing `'`

Reply:
I add the quotes higher up, but going to change it to binding, seems better that way overall.